### PR TITLE
clean up and update docs

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

clean up and update docs

## Issue reference

n/a

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: examples/demo_actor/README.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ export PYTHONPATH=`pwd`
 1. Run Demo Actor service in new terminal window
 
 ```bash
-$ cd examples/demo_actor
-$ dapr run --app-id demo-actor --app-port 3000 -- uvicorn --dapr-http-port 3000 demo_actor_service:app
+$ cd ./demo_actor
```
Comment (pruthvidhodda) [inline examples/demo_actor/README.md:39]: can we remove this

File: examples/demo_actor/README.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ dapr run --app-id demo-actor --app-port 3000 python3 demo_actor_flask.py
 1. Run Demo client in new terminal window
 
 ```bash
-$ cd examples/demo_actor
+$ cd ./demo_actor
```
Comment (pruthvidhodda) [inline examples/demo_actor/README.md:60]: we should remove this as well

Comment (youngbupark) [reply examples/demo_actor/README.md:60]: nope. demo_actor has demo_actor subdirectory.

File: examples/demo_actor/README.md (hunk 1 above)
Comment (youngbupark) [reply examples/demo_actor/README.md:39]: nope demo_actor has demo_actor sub-directory.

**Pull request merged**